README.md

```diff
@@ -30,4 +43,42 @@ Let's say Alice wants to swap 0.25 ETH into Stellar, and Bob wants to swap 0.25
 
 Additionally, both parties can refund their part of the swap after a certain (long) time period. This prevents malicious actors from stealing money, and other errors that would cause swaps to fail. For security reasons, the swap initiator (Bob) will have to wait longer to claim their refund than the swap fulfiller. Read more here: https://blog.lightning.engineering/announcement/2017/11/16/ln-swap.html
 
-Please let me know if I have overlooked anything, or if you have suggestions. Thanks!
\ No newline at end of file
+Please let me know if I have overlooked anything, or if you have suggestions. Thanks!
+
+
+Setup
+============
+
+## Prerequisites
+* `Docker`
+* `docker-compose`
+* `yarn`
+* `npm`
+
+
+## Environment Setup
+```bash
+# Clone the project
+git clone https://github.com/mvanderh/MirrorX.git
+cd MirrorX
+
+# Download Docker dependencies
+docker-compose build
```

Comment:
Don't need to `build`, just do `docker-compose up -d`

Reply:
`up` will start the container and `-d` will background it. Perhaps these are better suited for a "run" or "usage" section rather than an environment setup section? 

Up to you.

Reply:
Right. The only app running in Docker right now is just a local Redis, which doesn't need to be built. It would be better to move `docker-compose up -d` to usage like you said